packages/nimble-components/src/table/specs/table-column-width-hld.md

```diff
@@ -15,27 +15,45 @@ We need to provide users the means for changing the widths of individual columns
+
 -   Columns can be configured to not allow a user to interactively size them
     -   The implicit behavior present based on the behaviors described above, is that in a sizing action that cascades to a column configured to not be resized is that the column won't be resized towards a minimum size, and the cascade will effectively "skip" this column.
 
+#### Table Sizing
+
+-   Table resizing can result in _asymmetric_ column size state for mirrored actions such as a window maximize operation followed by a window restore down action. The following is a summary of expected behavior:
+    -   If the table currently has a horizontal scrollbar and the table is then sized to be wider, then as the table grows it will first remove excess space for the viewport prior to growing any columns. At this point, after the initial wider size has been completed, further reductions in window size will _not_ have a restorative state to the scrollbar prior to growing the window.
+    -   If the table currently has a horizontal scrollbar and the table is then sized to be smaller, the current proportional width of each column will be maintained.
+
+[Prototype branch](https://60e89457a987cf003efc0a5b-xtgmcptvna.chromatic.com/?path=/story/table-column-configuration--grouping).
+
 ### Out of Scope
 
 There are some column sizing behaviors that we will ultimately expect to support, but the APIs presented here are not meant to address:
 
 -   Auto-resizing: We will not describe how we intend to support the use-case of having a column auto-size to its contents
 -   Different interactive sizing modes: While the APIs described in this HLD do not inherently prescribe to a particular interactive sizing behavior, it's worth saying that in order to support multiple sizing modes, there will likely be additional APIs required that this HLD does not address.
--   Mechanisms related to accessibility-centric interactive column sizing (if there are such mechanisms). One possible example is allowing a user to size a column by way of the keyboard, instead of using a mouse. Ultimately, such a scenario is not in conflict with the API presented here, nor the mouse-based approach we know we will require, and can thus be handled separately, if ever.
+-   Mechanisms related to accessibility-centric interactive column sizing. One possible example is allowing a user to size a column by way of the keyboard, instead of using a mouse. Ultimately, such a scenario is not in conflict with the API presented here, nor the mouse-based approach we know we will require, and can thus be handled separately, if ever.
 
 ### API
 
-To accomodate the various sizing modes of a column, in addition to the other necessary behaviors, we can add the following properties to `ColumnInternals`:
+`Table`
+
+```ts
+    /**
+     * Allows a user to set how interactive column sizing should behave
+     */
+    @attr({ attribute: 'column-resize' })
+    public columnResize: 'none' | 'cascade' = 'cascade';
```

Comment:
nit: / implementation detail. The default value should be `undefined` (i.e. is not mandatory to be set) so if cascade is the default the enum should be:

```ts
export const TableColumnResize = {
    default: undefined,
    none: 'none'
}

```

Or potentially less proffered, naming the default option `cascade`, but that could cause potential confusion as the string `'cascade'` should not be an option:
```ts
export const TableColumnResize = {
    cascade: undefined,
    none: 'none'
}
```